package.json

```diff
@@ -11,16 +25,13 @@
     "node": ">=10"
   },
   "scripts": {
-    "start": "tsdx watch",
-    "build": "tsdx build",
-    "test": "tsdx test",
-    "lint": "tsdx lint",
-    "prepare": "tsdx build"
+    "start": "tsup --watch",
+    "build": "tsup",
+    "test": "true",
```

Comment:
I'm fine with dropping the linting steps, but could we keep the tests please? :)

Reply:
Oh dang, that was me in the last PR... forget about it 🤦